dffml/df/base.py

```diff
@@ -184,6 +192,45 @@ def op(*args, imp_enter=None, ctx_enter=None, config_cls=None, **kwargs):
     iterated over and the values in that ``dict`` are entered. The value yielded
     upon entry is assigned to a parameter in the ``OperationImplementation``
     instance named after the respective key.
+
+    Examples
+    --------
+
+    >>> from dffml import Definition, Input, op
+    >>> from typing import NamedTuple, List, Dict
+    >>>
+    >>> class Person(NamedTuple):
+    ...     name: str
+    ...     age: int
+    ...
+    >>> @op
+    ... def isTeen(p: List[Person]):
+    ...     return [True if person.age > 18 else False for person in p]
```

Comment:
```suggestion
    ...     return list(filter(lambda person: person.age < 18, p))
```

The `>` here should have been `<`